sharding-core/sharding-core-preprocessor/src/test/java/org/apache/shardingsphere/core/preprocessor/segment/select/projection/impl/ShorthandProjectionTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.core.preprocessor.segment.select.projection.impl;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public final class ShorthandProjectionTest {
+    
+    @Test
+    public void assertGetExpression() {
+        assertTrue(new ShorthandProjection("owner").getExpression().contains(".*"));
+    }
+    
+    @Test
+    public void assertGetAlias() {
```

Comment:
The method name should be `assertGetAliasWhenAbsent`
And here are missing assert for `assertGetAliasWhenPresent`

Reply:
hi, 
I think the name of assertGetAliasWhenAbsent is ok. 

but as for "here are missing assert for assertGetAliasWhenPresent", It seems that it is not necessary, as the whole method block under test is: 

```java
   @Override
    public Optional<String> getAlias() {
        return Optional.absent();
    }
```

so It seems that there is no possible the method would return Optional.present().

Reply:
OK